script/rspec-slow-repeat

```diff
@@ -15,7 +15,17 @@ fi
 
 n="$1"
 passed=0
-processors="`cat /proc/cpuinfo | grep -c processor`"
+
+# Check via uname the environment we are running in to get the number of cores
+if [[ "`uname`" == "Darwin" ]]; then
+  processors="`sysctl -n hw.ncpu.`"
+  env="mac"
```

Comment:
We don't really need this variable, do we?

Reply:
No that's not that important. That was mainly for debug.